### DISCUSSION TITLE
Usar Clean Architecture para ser um app escalavel

### DISCUSSION THREAD

**Discussion by Rodrigosiq03** [diego3g/rsxp-2023#28] [Ideas]
O Clean Architecture é uma arquitetura de código que faz com que a aplicação possa ter mudanças de framework, banco de dados e etc de uma maneira muito simples, sem precisar refatorar varios arquivos diferentes, com varias linhas de códigos, acho que usar o clean seria ótimo para o app, devido a vasta utilização das pessoas sobre o app. Creio que seja interessante utilizar o clean!!!

**Comment by nitoba** [comment]
Sim, é uma arquitetura que trás muita escalabilidade para o código, no entanto tem uma série de complexidades que que todos os devs não estão acostumados no dia a dia. Visto que esse é um projeto com um escopo menor, não seria necessário a utilização de clean architecture

**Comment by diego3g** [comment]
Fala pessoal, como falei na live hoje, quero manter o app o mais simples possível, totalmente "júnior friendly". Vamos manter a arquitetura o mais simples possível para que todo mundo possa ter segurança de contribuir 💜

Vou fechar essa discussão porque estamos com bastante discussões abertas com tópicos que super valem a contribuição de vocês por lá 🤗
